pages/privacy-and-permissions.tsx

```diff
@@ -0,0 +1,265 @@
+import Hero from "../components/Hero";
+import { useTranslations } from "next-intl";
+
+export async function getStaticProps({ locale }: { locale: any }) {
+  return {
+    props: {
+      metas: {
+        title:
+          "Privacy and Permissions policies, Open Source Day 2023 - Florence",
+        description:
+          "Open Source Day 2023 coming on the 24th of March 2023. Stay tuned on our social",
+      },
+      messages: (await import(`../public/locales/${locale}.json`)).default,
+    },
+  };
+}
+
+export default function CodeOfConduct() {
+  const t = useTranslations("PrivacyAndPermissions");
+  return (
+    <>
+      <div className="container">
+        <Hero title={t("title")} subtitle={t("subtitle")} originals={false} />
+        <div className="codeconduct">
+          <h3>Oggetto del trattamento</h3>
+
+          <p>
+            Informativa ai sensi dell&lsquo;art. 13 del Regolamento UE 679/2016. Ai
+            sensi dell&lsquo;art. 13 del Regolamento UE 679/2016, recante disposizioni
+            a tutela delle persone e di altri soggetti rispetto al trattamento
+            dei dati personali, desideriamo informarLa che i dati personali da
+            Lei forniti formeranno oggetto di trattamento nel rispetto della
+            normativa sopra richiamata.
+          </p>
+
+          <h3>Titolare</h3>
+
+          <p>
+            Titolare del trattamento è l&lsquo; Associazione non riconosciuta
+            Schrodinger Hat, in persona dei soci Miki Lombardi, Gabriele
+            Puliti, Nicola Puppa, Davide Imola, Lorenzo Pieri (in seguito,
+            “Titolare”). L&lsquo; Associazione non riconosciuta Schrodinger Hat,
+            provvede a svolgere attività didattica in ambito di progettazione
+            software e di organizzazione di corsi formativi e di istruzione.
+          </p>
+
+          <h3>Tipologia di dati trattati</h3>
+
+          <p>
+            In relazione a tali attività professionali saranno necessariamente
+            trattati i Vostri dati comuni (a titolo esemplificativo e non
+            esaustivo: nome, cognome, luogo di nascita, codice fiscale, luogo di
+            residenza, ecc.). Questi dati saranno trattati dal titolare del
+            trattamento per espletamento dell&lsquo;attività professionale richiesta
+            nel rispetto della normativa vigente in materia di protezione dei
+            dati.
+          </p>
+
+          <h3>Dati personali di Suoi familiari</h3>
+
+          <p>
+            Per la gestione dell&lsquo;incarico professionale ricevuto il Titolare
+            potrà, altresì, trattare dati personali di Vostri familiari al fine
+            di procedere all&lsquo;emissione della corretta intestazione della fattura
+            per il pagamento della prestazione erogata dall&lsquo;azienda. In tal
+            senso Vi impegnate ad informare i Vostri familiari di tali
+            trattamenti, anche fornendo loro la presente informativa.
+          </p>
```

Comment:
Eviterei di inserire questo dato che non è relativo all'evento e alla gestione dei dati in questione